### ISSUE TITLE
tishadow constantly disconnects

### ISSUE THREAD

**Issue by icecandy** [dbankier/TiShadow#549]
I've been using tishadow for quite a while and generally works fine. I have a specific annoying problem that about 50% of the time tishadow DISCONNECTS upon a save and then I have to recompile.
So compiling with express using the tishadow flag and the following at cli:
$ appc ti build -p ios --target simulator --device-id A88F7442-E023-4B1E-8646-B3196EDC8285 --shadow --host 192.168.0.24
The disconnect seems random, at least I can't see a pattern in when it happens.
it just kind of defeats the whole point of using tishadow if 50% of the time it doesn't work.

**Comment by dbankier** [comment]
Can you share any logs you have? Mine is smooth.

**Comment by icecandy** [comment]
Not sure what logs you mean. In the terminal I just get:
$ [WARN] [iphone, 11.3, undefined] Disconnected
It just happened now and actually not even on a save. It sometimes randomly disconnects after a certain length of time - maybe a few minutes - even though I am not using the app - eg go away for lunch, come back and it has disconnected.

**Comment by dbankier** [comment]
Are you still getting this issue?

**Comment by icecandy** [comment]
yes this is still happening. Currently using Ti SDK 7.5.0. iOS and Android.
